src/providers/bettersqlite.js

```diff
@@ -0,0 +1,239 @@
+const SettingProvider = require('./base');
+
+/**
+ * Uses an SQLite database to store settings with guilds
+ * @extends {SettingProvider}
+ */
+class BetterSQLiteProvider extends SettingProvider {
```

Comment:
I'd prefer it if the provider were called something else, like "SyncSQLiteProvider". The filename should be `sqlite-sync.js`, too.